spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -1526,10 +1526,12 @@ Obtaining the next 10 products that cost $50.00 or more:
 [source,java]
 ----
 float priceMidpoint = 50.0f;
-PageRequest<Product> pageRequest = Order.by(_Product.price.asc(), _Product.id.asc())
-                                        .pageSize(10)
-                                        .afterKey(priceMidpoint, 0L);
-CursoredPage<Product> moreExpensive = products.findByNameLike(pattern, pageRequest);
+PageRequest<Product> pageRequest =
```

Comment:
Why not include the extra space here as you did at:
https://github.com/jakartaee/data/pull/590/files#diff-451947eabf993881db66ea833734022a76dff66ae9b0d26c2417e187fa3932acR1520